specs/PushNotifications/PushNotifications-spec.md

```diff
@@ -459,12 +592,21 @@ namespace Microsoft.Windows.PushNotifications
         // Unregister any activator if present using a token and registrationKind
         // 1) If kind = PushTrigger is specified, the trigger itself will be removed
         // 2) If kind = ComActivator is specified, the Project Reunion Background Task component will no longer act as an InProc COM Server
-        static void UnregisterActivator(PushNotificationRegistrationToken token, PushNotificationRegistrationKind kind);
+        // 3) If kind = ProtocolActivator is specified, the application will be unregistered from the PushNotificationsLongRunningTask
+        static void UnregisterActivator(PushNotificationRegistrationToken token, PushNotificationRegistrationOptions options);
 
         // Request a Push Channel with an encoded RemoteId from WNS. RemoteId is an AAD identifier GUID
         static Windows.Foundation.IAsyncOperationWithProgress<PushNotificationCreateChannelResult, PushNotificationCreateChannelStatus> CreateChannelAsync(Guid remoteId);
+
+        // Developers should call this to know with which PushNotificationRegistrationKind to call RegisterActivator. 
+        // Packaged applications should set the isInProcActivation to true if they want the current process to handle background activation. 
+        // Unpackaged applications should always set the isInProcActivation to false.
```

Comment:
Why? Out-of-proc is unsupported for unpackaged apps.

Reply:
Hmm can we support it for unpackaged?